import sys

def error_exit(error_msg):
	print("Error: {}".format(error_msg))
	sys.exit()
